doc/Plugins.md

```diff
@@ -0,0 +1,127 @@
+## BMX7 Plugins ##
+
+### Compile and Install ###
+
+To compile and install bmx7 daemon and all bmx7 plugins simply do:
+<pre>
+make build_all EXTRA_CFLAGS="-DTRAFFIC_DUMP -DCRYPTLIB=MBEDTLS_2_4_0"
+sudo make install_all
+</pre>
+
+However. specific requirements may need to be fulfilled for some plugins in order to compile correctly.
+These requirements are described in the corresponding plugin section.
+
+### Config Plugin ###
+
+#### Requirements ####
+
+uci libs are needed for the bmx7-config plugin.
+To install try (old version):
+<pre>
+wget http://downloads.openwrt.org/sources/uci-0.7.5.tar.gz
+tar xzvf uci-0.7.5.tar.gz
+cd uci-0.7.5
+make clean all install WOPTS="-pedantic -Wall"
+sudo make install
+</pre>
+
+
+#### Compile and Install ####
```

Comment:
Stick to correct markdown, without the trailing hashtags

Reply:
Done